workers/social/lib/social/models/group/index.coffee

```diff
@@ -1705,6 +1705,12 @@ module.exports = class JGroup extends Module
         JName.one { name: slug }, (err, name) ->
           removeHelper name, err, next, 'JName'
 
+      (next) ->
+        # unlink slack if anyone in this team auth slack previously
+        JUser = require '../user'
+        JUser.update { "foreignAuth.slack.#{slug}": { $exists: true } }, { $unset: { "foreignAuth.slack.#{slug}": 1 } }, (err) ->
+          kallback 'JForeignAuth', next, err
```

Comment:
we no longer have `JUser.foreignAuth.*`[1] in the schema. It seems the slack integration is still using that, it's ok to cleanup them but it would be nice if we can do it over `socialApi` since it's added by that. cc/ @cihangir 

[1] https://github.com/koding/koding/pull/9057